src/service/viewer-impl.js

```diff
@@ -844,7 +853,7 @@ export class Viewer {
     if (!this.hasCapability('fragment')) {
       return Promise.resolve('');
     }
-    return this.sendMessageUnreliable_('fragment', undefined, true).then(
+    return this.sendMessageCancelUnsent('fragment', undefined, true).then(
```

Comment:
This should be switched to `getMessage` right away. "cancel unsent" really wrong here.